### ISSUE TITLE
Raw string in pdf when used with Django SafeText 

### ISSUE THREAD

**Issue by jgsogo** [romanlv/trml2pdf#5]
I've implemented a tiny example with Django in order to test trml2pdf using the template system. Here you can checkout a [minimal working example](https://github.com/jgsogo/trml2pdf-issue). I'm using Python 3.4.3 and Win7 64 bits.

Generated PDF contains raw string in the paragraph (see actual pdf [here](https://github.com/jgsogo/trml2pdf-issue/blob/master/report.pdf)):

```
b'\n Unicode aqu\xc3\xad.\n y aqu\xc3\xad.\n Hello Mar\xc3\xada\n '
```

and expected should be (not sure about blanks and newlines):

```
 Unicode aquí
               y aquí.
 Hello María
```

View class is implemented as follows:

``` python
class PDFView(View):
    template_name = 'rml/template.rml'

    def get_template_name(self):
        if self.template_name is None:
            raise ImproperlyConfigured('%s requires either a definition of '
                                       'template_name or an implementation of '
                                       'get_template_name()'
                                       % self.__class__.__name__)
        return self.template_name

    def get_context_data(self, **kwargs):
        return {'filename': 'report.pdf',
                'user': 'María'}

    def get(self, request, *args, **kwargs):
        context = self.get_context_data(**kwargs)
        context.setdefault('filename', 'report.pdf')

        rml = render_to_string(self.get_template_name(), context)  # Type of 'rml' is django.utils.safestring.SafeText
        response = HttpResponse(content_type='application/pdf')
        response['Content-Disposition'] = ('filename="report.pdf"')
        response.write(trml2pdf.parseString(rml))
        return response
```

**Comment by willson556** [comment]
Hi,

I changed some of the text handling code on our fork, see if that helps. I've been planning to put in another pull request but wanted to finish up a few things first.

https://github.com/CalPolyResDev/trml2pdf

**Comment by jgsogo** [comment]
Hi, sorry for the delay. I won't be using this package for this project anymore (we are building the PDF right in code using reportlab), nevertheless I can keep the issue opened and try to fix it and make the PR.

The code you said you have changed, may I expect a pull-request?

**Comment by romanlv** [comment]
should be fixed with #10
